### PR TITLE
Adding Python to OData modal and changing text

### LINKED ISSUE #740: Add pyODK to OData modal
1. Change the initial paragraph to the text below. Note that Power BI is first, and we've added a link for Excel.
> OData is a standard for transferring data between tools and services. Powerful analysis tools like Power BI (https://powerbi.microsoft.com/en-us/), Excel (https://www.microsoft.com/en-us/microsoft-365/excel), Python (https://www.python.org/), and R (https://www.r-project.org/) can fetch data via OData for analysis.
> 
> To connect to this form's OData feed, select your tool and copy the link into it.

2. Add a Python pane before R panel with the following blurb.
> To connect to Central from Python, we recommend pyODK (https://github.com/getodk/pyodk/). pyODK is the official Python client for Central and it makes common data analysis and workflow automation tasks simple.

3. Simplify R blurb
> To connect to Central from R, we recommend ruODK (https://docs.ropensci.org/ruODK/). ruODK is developed and supported by ODK community members.

4. Change "Excel/Power BI" to "Power BI or Excel" on the header. Looks better to me, and I think Power BI should be first.

5. Update Excel link in blurb.
> For help using OData with Power BI, see this page (https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-connect-odata). For help with Excel, see this page (https://support.microsoft.com/en-us/office/about-power-query-in-excel-7104fbee-9e62-4cb9-a02e-5bfb1a6c536a).

### PR DESCRIPTION
**Pull request by ktuite**
Closes #740. This will show up on the submissions page and the entities page. 

#### What has been done to verify that this works as intended?

Tests and looking at it. 

Screen Shot 2023-02-24 at 11 25 22 AM

#### Why is this the best possible solution? Were any other approaches considered?

Pretty straightforward template and text changes following guidance in issue.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

Adds python! Less text. 

#### Does this change require updates to user documentation? If so, please file an issue [here](https://github.com/getodk/docs/issues/new) and include the link below.

No.

#### Before submitting this PR, please make sure you have:

- [x] run `npm run test` and `npm run lint` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

File: src/components/submission/analyze.vue [hunk 1]
```diff
@@ -213,272 +200,3 @@ export default {
 }
 </i18n>
 
-<!-- Autogenerated by destructure.js -->
```
Comment (matthew-white) [inline src/components/submission/analyze.vue:216]: Probably no need to delete this section even if things are broken for non-English locales, unless tests are failing. I'll update this section when I update translations before regression testing.

File: src/components/submission/analyze.vue [hunk 2]
```diff
@@ -176,34 +173,24 @@ export default {
...
     "introduction": [
       // The text of {powerBi} is "Microsoft Power BI". The text of {r} is "R".
-      // {powerBi} and {r} are both links.
-      "OData is a new standard for transferring data between tools and services. Free and powerful analysis tools like Excel, {powerBi}, and {r} can fetch data over OData for analysis.",
-      "There are many advantages to OData, but importantly it supports the full-fidelity transfer of complicated types like numbers and geographic data, and it enables the latest version of your data to synchronize easily with any tools using it.",
-      "To begin using OData, select your tool and copy the link into it."
+      // The text of {python} is "Python". The text of {excel} is "Excel".
+      // {powerBi}, {r}, {excel} and {python} are all links.
+      "OData is a standard for transferring data between tools and services. Free and powerful analysis tools like {powerBi}, {excel}, {python}, and {r} can fetch data via OData for analysis.",
+      "To connect to this form’s OData feed, select your tool and copy the link into it."
```
Comment (matthew-white) [inline src/components/submission/analyze.vue:179]: form → Form

File: src/components/submission/analyze.vue [hunk 3]
```diff
@@ -176,34 +173,24 @@ export default {
...
-          // {oData} is "OData". All three are links.
-          "full": "To access Central data from {r}, we recommend you use {ruODK}. See ruODK’s vignettes for examples of using both the {oData} and the {restful} API.",
-          "restful": "RESTful"
-        },
-        {
-          "full": "Just like ODK itself, ruODK is developed and supported by community members. If you wish to help improve it, you can find information {here}.",
-          "here": "here"
-        }
-      ],
+      "python": "To connect to Central from Python, we recommend {pyODK}. pyODK is the official Python client for Central and it makes common data analysis and workflow automation tasks simple.",
```
Comment (matthew-white) [inline src/components/submission/analyze.vue:192]: How about a translator comment here:

> {pyODK} is a link. Its text is "pyODK".

File: src/components/submission/analyze.vue [hunk 4]
```diff
@@ -176,34 +173,24 @@ export default {
...
+      "r": "To connect to Central from R, we recommend {ruODK}. ruODK is developed and supported by ODK community members.",
```
Comment (matthew-white) [inline src/components/submission/analyze.vue:193]: Probably a translator comment here as well

File: src/components/submission/analyze.vue [hunk 5]
```diff
@@ -45,33 +53,22 @@ except according to the terms contained in the LICENSE file.
...
+            <a href="https://support.microsoft.com/en-us/office/about-power-query-in-excel-7104fbee-9e62-4cb9-a02e-5bfb1a6c536a" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForExcel') }}</a>
           </template>
           <template #pageForPowerBi>
-            <a href="https://docs.microsoft.com/en-us/power-bi/desktop-connect-odata" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForPowerBi') }}</a>
+            <a href="https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-connect-odata" target="_blank" rel="noopener">{{ $t('help.microsoft.pageForPowerBi') }}</a>
+          </template>
+        </i18n-t>
+        <i18n-t v-else-if="tool === 'python'" tag="p" keypath="help.python">
+          <template #pyODK>
+            <a href="https://github.com/getodk/pyodk/" target="_blank" rel="noopener">PyODK</a>
```
Comment (matthew-white) [inline src/components/submission/analyze.vue:64]: PyODK → pyODK? How do we stylize the name?

**Review by matthew-white — APPROVED** [review]

File: src/components/submission/analyze.vue (hunk 5 above)
Comment (ktuite) [reply src/components/submission/analyze.vue:64]: Ah yea, it should just be pyODK.

**Pull request merged**